src/components/ForgotPassword/ForgotPassword.js

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from 'react';
+import {
+  Redirect,
+} from 'react-router-dom';
+
+import processBadResponse from '../../processBadResponse';
+import { defaultErrors } from '../../constants';
+import ForgotPasswordForm from './ForgotPasswordForm';
+import useForgotPasswordMutation from './useForgotPasswordMutation';
+
+const ForgotPassword = () => {
+  const [userEmail, setUserEmail] = useState(null);
+  const [authFailure, setAuthFailure] = useState([]);
+  const sendReminderMutation = useForgotPasswordMutation();
+
+  const onSubmit = async (values) => {
```

Comment:
it would be great to have loading/smth to show request is sent, our QAs like to press button many times :)

Reply:
it's more like enhancement, previous implementation is the same